pkg/fleet-manager/pipeline/render/predefined_task.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package render
+
+import (
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	pipelineapi "kurator.dev/kurator/pkg/apis/pipeline/v1alpha1"
+)
+
+const (
+	// GitCloneTask is the predefined task template name of git clone task
+	GitCloneTask = "git-clone"
+	// GoTestTask is the predefined task template name of go test task
+	GoTestTask = "go-test"
+	// GoLintTask is the predefined task template name of golangci lint task
+	GoLintTask = "go-lint"
+	// BuildPushImage is the predefined task template name of task about building image and pushing it to image repo
+	BuildPushImage = "build-and-push-image"
```

Comment:
use that in type.go

Reply:
In my recollection, I have already made changes to this part after I add them all in type.go. 

It's possible that the wrong branch was chosen. 

I'll make some adjustments.